src/main/java/tk/shanebee/hg/game/GamePlayerData.java

```diff
@@ -305,6 +307,20 @@ public void join(Player player, boolean command) {
                 freeze(player);
                 kills.put(player, 0);
 
+                ItemStack bed = new ItemStack(Material.RED_BED, 1);
+                ItemMeta commeta = bed.getItemMeta();
+                commeta.setDisplayName("Leave");
+                bed.setItemMeta(commeta);
+                player.getInventory().setItem(8, bed);
+
+                if (player.isOp()){
```

Comment:
2 things here:
1) there should be a space between the last parenthesis and the bracket
2) rather than an OP check, there should be a permission check. I can't think of the appropriate permission off the top of my head, but probably whatever permission would allow the player to use the force start command